libnavigation-core/src/main/java/com/mapbox/navigation/core/formatter/FormattedDistanceData.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.core.formatter
+
+import com.mapbox.navigation.base.formatter.UnitType
+
+/**
+ * Represents a distance that has been formatted and potentially rounded for display purposes.
+ *
+ * @param distance the calculated distance value
+ * @param distanceAsString a rounded string representation of the distance. For example a distance value of 19.3121 might result in a string value of '19'
+ * @param distanceSuffix a suffix that goes with the string value of the distance like 'km' for kilometers, 'ft' for foot/feet, 'mi' for miles etc. depending on the Locale and the UnitType
+ * @param unitType indicates if the values represented are metric or imperial
+ */
+data class FormattedDistanceData(
```

Comment:
should be `internal` constructor? if so need to remove `data` and override quals/toString/hashCode